src/sentry/auth/access.py

```diff
@@ -237,10 +330,10 @@ def has_project_access(self, project: Project) -> bool:
         return True
 
     def has_project_membership(self, project: Project) -> bool:
-        return True
+        return False
```

Comment:
Out of curiosity, why is this false now?

Reply:
Along the same lines as https://github.com/getsentry/sentry/pull/32086#issuecomment-1057223992. It's for consistency with an `OrganizationGlobalAccess` for a Sentry app or global auth, and otherwise to just correctly do what the docstring says. A `SystemAccess` object can access all projects but it not specifically signed up as a member of anything.

AFAIK this affects nothing.